march_hardware/src/IMotionCube.cpp

```diff
@@ -155,6 +155,15 @@ float IMotionCube::getAngleRad()
   return this->encoder.getAngleRad(this->misoByteOffsets[IMCObjectName::ActualPosition]);
 }
 
+float IMotionCube::getTorque()
+{
+  ROS_ASSERT_MSG(this->misoByteOffsets.count(IMCObjectName::ActualTorque) == 1, "ActualTorque not defined in PDO "
+                                                                                "mapping, so can't get angle");
```

Comment:
Change to "can't get torque"

Reply:
https://github.com/project-march/hardware-interface/pull/130